### PR TITLE
[bitnami/memcached] Add optional emptyDir medium configuration

### PR DESCRIPTION
**Pull request by nhh**
### Description of the change

This change adds the ability to define the emptyDir configuration of memcache to be "Memory"

### Benefits

People can decide to configure their tmp directory to be tmpfs (RAM-backed filesystem)

### Possible drawbacks
See k8s docs for memory impact on tmpfs volumes: https://kubernetes.io/docs/concepts/storage/volumes/#emptydir

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/memcached/Chart.yaml [hunk 1]
```diff
@@ -31,4 +31,4 @@ maintainers:
 name: memcached
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/memcached
-version: 7.4.14
+version: 7.4.15
```
Comment (jotamartos) [inline bitnami/memcached/Chart.yaml:34]:
```suggestion
version: 7.5.0
```

As you are including new parameters, you need to bump the version to a new minor one

**Review by jotamartos — CHANGES_REQUESTED** [review]
The change looks good to me. However, I think you also need to apply the change in the templates/statefulset.yaml file.

**Comment by nhh** [comment]
> The change looks good to me. However, I think you also need to apply the change in the templates/statefulset.yaml file.

@jotamartos thx for having a look, I added your suggestions!

**Comment by nhh** [comment]
There were merge conflicts due to an automatic update happening in the mean time using my previous used chart version 7.4.15, which I did not know about. I thought it was the bot hallucinating and tried to remove it from the changelog.

I now need to to everything all over again

**Pull request closed without merging**